public/src/modules/translator.js

```diff
@@ -295,11 +295,13 @@
 	if (typeof define === 'function' && define.amd) {
 		define('translator', translator);
 
+		var _translator = translator;
+
 		// Expose a global `translator` object for backwards compatibility
 		window.translator = {
 			translate: function() {
 				console.warn('[translator] Global invocation of the translator is now deprecated, please `require` the module instead.');
-				translator.translate.apply(translator, arguments);
```

Comment:
Perhaps I'm being thick, but I don't understand the change here :frowning: `translator` is a reference to the module object, and assigning `_translator` just also sets that reference, so nothing actually changed?

Edumacate me please :smile: